eng/scripts/Language-Settings.ps1

```diff
@@ -240,15 +236,15 @@ function Find-javascript-Artifacts-For-Apireview($artifactDir, $packageName = ""
   return $packages
 }
 
-function SetPackageVersion ($PackageName, $Version, $ServiceDirectory = $null, $ReleaseDate, $BuildType = $null, $GroupId = $null)
+function SetPackageVersion ($ArtifactName, $Version, $ServiceDirectory = $null, $ReleaseDate, $BuildType = $null, $GroupId = $null)
 {
   if ($null -eq $ReleaseDate)
   {
     $ReleaseDate = Get-Date -Format "yyyy-MM-dd"
   }
   Push-Location "$EngDir/tools/versioning"
   npm install
-  node ./set-version.js --artifact-name $PackageName --new-version $Version --release-date $ReleaseDate --repo-root $RepoRoot
+  node ./set-version.js --artifact-name $ArtifactName --new-version $Version --release-date $ReleaseDate --repo-root $RepoRoot
```

Comment:
Consider doing a check to see whether the node command is available in advance (like discussed on the Teams channel).